src/main/java/de/eldoria/pickmeup/config/MobSettings.java

```
@@ -54,6 +54,7 @@ public MobSettings() {
     }
 
     public boolean canBePickedUp(Player player, EntityType type) {
+        if (allowedMobs == null) return false;
```

While I appreciate the check, I would rather throw an error at the user that they might need to check their config than silently fail. Exceptions are not there to be avoided all the time. If the plugin is misconfigured they should be alerted